client/extensions/woocommerce/state/action-list/selectors.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Get the current step in the action list.
+ *
+ * @param {Object} rootState The root Calypso state.
+ * @return {Array|null} The action list, or null if no action list is present.
+ */
+export function getActionList( rootState ) {
+	const woocommerce = rootState.extensions.woocommerce;
+	return get( woocommerce, 'actionList', null );
```

Comment:
I started doing this because `state.extensions.woocommerce` should always exist, it's only from that point on that we're not sure if the path exists. Also, if we ever end up in another top-level redux structure, the top line could change without having to change the rest of it.

i.e. I had a reason for doing it, but I'm willing to change this if the team feels strongly about it.